utils/sccli.sh

```diff
@@ -68,7 +68,23 @@ function usage()
 
 function openshift()
 {
+    # Default OpenShift Image and Tag
+    dockerRegistry=${DOCKER_REGISTRY:-docker.io}
+    imageName=${IMAGE_NAME:-openshift/origin}
+    imageTag=${IMAGE_TAG:-v1.1.1}
+
     clean_setup
+
+    # Setup the systemd environment - desired image may have changed
+    sudo sed -i.back "/^IMAGE=*/cIMAGE=\"${imageName}\:${imageTag}\"" /etc/sysconfig/openshift_option
+
+    # ensure the docker containers are present
+    docker pull ${dockerRegistry}/${imageName}:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-haproxy-router:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-deployer:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-docker-registry:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-sti-builder:${imageTag}
+
```

Comment:
@praveenkumar Agree that docker pulls are smart. But IMO we should not try docker pull when images are present. Because we do not want to get in any docker pull issues (I am just being defensive here) . I have couple of incidents when docker pull sometimes did not work as expected (may be because of of my docker environment)